edweiss-app/app/(app)/startCourseScreen/index.tsx

```diff
@@ -65,9 +65,9 @@ const StartCourseScreen: ApplicationRoute = () => {
 	useEffect(() => {
 		setColorButtonStartStop(parsedCourse.started ? "transparent" : "overlay2");
 		setTextButton(parsedCourse.started ? "Stop course" : "Start course");
-		setColorTextStartStop(parsedCourse.started ? "flamingo" : "course_title_for_backgroud_color");
+		setColorTextStartStop(parsedCourse.started ? "flamingo" : "constantBlack");
 		setTextButtonShow(available ? "Sharing in progress..." : "Show to student");
-		setColorTextShow(available ? "flamingo" : "course_title_for_backgroud_color");
+		setColorTextShow(available ? "flamingo" : "constantBlack");
 		setColorButtonShow(available ? "transparent" : "overlay2");
```

Comment:
You can simplifie this code by exemple : 

```
const getButtonStyles = () => ({
  startStopButton: {
    backgroundColor: parsedCourse.started ? "transparent" : "overlay2",
    text: parsedCourse.started ? "Stop course" : "Start course",
    textColor: parsedCourse.started ? "flamingo" : "constantBlack",
  },
  showButton: {
    backgroundColor: available ? "transparent" : "overlay2",
    text: available ? "Sharing in progress..." : "Show to student",
    textColor: available ? "flamingo" : "constantBlack",
  },
});

const { startStopButton, showButton } = getButtonStyles();
```

Reply:
This is not my own code, I've only modified the name of colour into "constantBlack" since Youssef told me it was a problem to have too many colours